src/services/config/index.ts

```diff
@@ -74,11 +74,20 @@ class ConfigService {
 
   private readonly configFileMustExist = (): void => {
     const versionConf = this.store.getVersion()
-    if (versionConf === null) {
+    if (versionConf === null || versionConf === undefined) {
```

Comment:
isn't better to use the shorthand `if (!versionConf)`  here?

Reply:
we thought to be more explicit to prevent it from passing for an empty string `""` and zero `0`